src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -57,13 +57,13 @@
     @ArgumentCollection
     protected IntervalArgumentCollection intervalArgumentCollection = requiresIntervals() ? new RequiredIntervalArgumentCollection() : new OptionalIntervalArgumentCollection();
 
-    private JavaSparkContext sparkContext;
-    private ReadsSparkSource readsSource;
-    private SAMFileHeader readsHeader;
+    private transient JavaSparkContext sparkContext;
+    private transient ReadsSparkSource readsSource;
+    private transient SAMFileHeader readsHeader;
     private String readInput;
-    private ReferenceMultiSource referenceSource;
-    private SAMSequenceDictionary referenceDictionary;
-    private List<SimpleInterval> intervals;
+    private transient ReferenceMultiSource referenceSource;
+    private transient SAMSequenceDictionary referenceDictionary;
+    private transient List<SimpleInterval> intervals;
```

Comment:
True, but serialization should only happen when this is shipped to the workers and workers should never be calling those functions. If you feel strongly I can limit this to just the `JavaSparkContext`.